packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -452,6 +449,29 @@ const toggleOrientation = async ( driver ) => {
 	}
 };
 
+const isEditorVisible = async ( driver, iteration = 0 )  => {
+	if ( iteration >= 25 ) {
+		throw new Error(
+			`Gutenberg Editor is still not visible after ${ iteration } retries!`
+		);
+	} else if ( iteration !== 0 ) {
+		// wait 1 second before trying to locate element again
+		await driver.sleep( 1000 );
+	}
```

Comment:
I'm wondering if we could have either the iterations limit and wait time as constants, wdyt?